### ISSUE TITLE
bug: Spelling error "fund" should be "found"

### ISSUE THREAD

**Issue by algoritmoalex** [kubestellar/kubestellar#2216]
### Describe the bug

Wording needs to be fixed. "Fund" should be "found" in the link below.

https://github.com/kubestellar/kubestellar/blob/release-0.23.0/docs/content/direct/core-chart.md

### Steps To Reproduce

1. Go to link: https://github.com/kubestellar/kubestellar/blob/release-0.23.0/docs/content/direct/core-chart.md
2. Go to Pre-Requisites
3. End of first paragraph there is a spelling error of "found".

TL;DR - Ctrl+F "Fund" to find the problem.

### Expected Behavior

Correct spelling.

**Comment by clubanderson** [comment]
@algoritmoalex can you see if this is still an issue?

/assign @algoritmoalex
